zephyr/boot_test.c

```diff
@@ -4,9 +4,19 @@
  */
 
 #include <zephyr/logging/log.h>
+#include <sof/boot_test.h>
 
 #include <zephyr/ztest.h>
 
 LOG_MODULE_REGISTER(sof_boot_test, LOG_LEVEL_DBG);
 
 ZTEST_SUITE(sof_boot, NULL, NULL, NULL, NULL, NULL);
+
+#if CONFIG_SOF_BOOT_TEST
```

Comment:
this isn't needed - this file isn't built if `CONFIG_SOF_BOOT_TEST` isn't selected